runelite-client/src/main/java/net/runelite/client/account/SessionManager.java

```diff
@@ -232,9 +234,17 @@ public void onLogin(LoginResponse loginResponse)
 		saveSession();
 	}
 
+	@Inject
+	RuneLite runeLite;
+
 	public void logout()
 	{
-		closeSession();
-		deleteSession();
+		if (JOptionPane.YES_OPTION == JOptionPane.showConfirmDialog(runeLite.getGui(),
```

Comment:
Don't do this. It is perfectly fine to set "null" as parent for dialog, no need to inject RuneLite and call getGui on it.

Reply:
It was so the new dialog would be centered and relative to RuneLite's location